apps/vmq_server/src/vmq_server_cli.erl

```diff
@@ -95,6 +95,7 @@ register_cli_usage() ->
     clique:register_usage(["vmq-admin", "api-key"], api_usage()),
     clique:register_usage(["vmq-admin", "api-key", "delete"], api_delete_key_usage()),
     clique:register_usage(["vmq-admin", "api-key", "add"], api_add_key_usage()),
+    clique:register_usage(["vmq-admin", "api-key", "create"], api_create_key_usage()),
```

Comment:
@mths1 why not just have the "add" command?

Reply:
Ah, never mind, we seemed to make a difference between "add" and "create" before too.